rpcclient/chain.go

```diff
@@ -267,6 +270,32 @@ func (r FutureGetBlockChainInfoResult) Receive() (*btcjson.GetBlockChainInfoResu
 	if err := json.Unmarshal(res, &chainInfo); err != nil {
```

Comment:
How will this handle the new fields? Could be worth extracting this logic into a method that we can add unit tests for.

Reply:
They're both nil since they don't have JSON tags.

```
(btcjson.GetBlockChainInfoResult) {
 Chain: (string) (len=7) "regtest",
 Blocks: (int32) 423,
 Headers: (int32) 423,
 BestBlockHash: (string) (len=64) "6d23bb7688d0e51e81cf5003d99943001c410650c16748f2a3a6f9e1d0d833d2",
 Difficulty: (float64) 4.656542373906925e-10,
 MedianTime: (int64) 1572545865,
 VerificationProgress: (float64) 1,
 Pruned: (bool) false,
 PruneHeight: (int32) 0,
 ChainWork: (string) (len=64) "0000000000000000000000000000000000000000000000000000000000000350",
 SoftForks: (*btcjson.SoftForks)(<nil>),
 UnifiedSoftForks: (*btcjson.UnifiedSoftForks)(<nil>)
}
```

Reply:
Would be nice with a testcase :)

Reply:
Done.